spec/classroom_spec.rb

```diff
@@ -0,0 +1,13 @@
+require_relative '../classroom'
+
+describe Classroom do
+  before do
+    @classroom = Classroom.new('English')
+  end
+
+  describe '#initialize' do
+    it 'initializes with a label' do
+      expect(@classroom.label).to eq('English')
+    end
+  end
+end
```

Comment:
- [ ] Great job so far 👏🏻 👏🏻 However, according to the project requirements, there should be unit tests for all the methods in all the classes, Kindly make sure the `add_student` class classroom method has (at least) an 'it' block 👇🏻 check your classroom.rb file

```ruby
  def add_student(student)
    @students << student
    student.classroom = self
  end
```